pipenv.el

```diff
@@ -289,6 +289,21 @@ or (if none is given), installs all packages."
                        real-path)))
     (find-file ideal-path)))
 
+(defun pipenv--messaging-sentinel (process event)
+  "Send EVENT notifications for PROCESS to *Messages* buffer."
+  (message "Process %s: event %S" process (s-chomp event)))
+
+(defun pipenv--check-output (&rest command)
```

Comment:
It is great that you have gotten rid of the shell command hacks and reused `make-pipenv-process`. Thanks.

Reply:
I wouldn't mind renaming this to something other than `check-output`, but I'll accept it as it is and possibly refactor in a later commit.